membership_policy.md

```diff
@@ -2,13 +2,15 @@
-| Reviewer | Review contributions from others                                                                             | * Member<br>* History of review and authorship in the project<br> * Sponsored by an approver | [OWNERS_ALIASES] file reviewer entry                                                          |
-| Approver| Approve accepting contributions                                                                              | * Reviewer<br>* Highly experienced<br> * Active reviewer & contributor to the project<br>Sponsored by 2 approvers| [OWNERS_ALIASES] file approver entry |
-
+| Role                                           | Responsibilities                                                                                             | Requirements                                                                                                         | Defined by                           |
+|------------------------------------------------|--------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+| [Contributor](#new-contributors)               | Adhere to the [KubeVirt code of conduct](https://github.com/kubevirt/community/blob/main/code-of-conduct.md) | Submit at least one pull request                                                                                     | Active GitHub account                |
+| [Member](#member)                              | Active contributor in the community                                                                          | * Active contributor<br>* Sponsored by 2 members<br>* Multiple contributions to the project                          | KubeVirt GitHub org member           |
+| [Reviewer](#reviewer)                          | Review contributions from others                                                                             | * Member<br>* History of review and authorship in the project<br> * Sponsored by an approver                         | [OWNERS_ALIASES] file reviewer entry |
+| [Approver](#approver)                          | Approve accepting contributions                                                                              | * Reviewer<br>* Highly experienced<br> * Active reviewer & contributor to the project<br> * Sponsored by 2 approvers | [OWNERS_ALIASES] file approver entry |
+| [SIG Chair](#special-interest-group-sig-chair) | Lead a SIG aligned to the goals of the SIG charter                                                           | * sig-approver<br>* Highly experienced in SIG matters<br> * Active reviewer & contributor to the project             | [sigs.yaml] chair entry              |
```

Comment:
In the section for the SIG chair, the requirement does not mention `sig-approver` but here it is.
And I think we checked this with Kubernetes requirements, which does not mention such a requirement from a chair.

Reply:
@EdDev I have just noticed the paragraph below https://github.com/kubevirt/community/pull/306/files#diff-78a4ac3dbf91e4f4ac17004bc8e0bca8009fcd12e09d46da4b9ab11848e4089aR157

And I think being an approver for the sig as a sig chair does make sense. WDYT?

Reply:
It does to me.

Reply:
> > Claims approver rights in scope
> 
> I read that point based on its content sub-points. I understood it as being responsible to have something like that in a project, not necessarily to also be the one who approves the changes.
> 
> OK, so the suggestion is to couple the chair [1] and tech-lead [2] roles together. Right? The added value of differentiating them is that the chair can administratively manage a SIG while the lead can just take technical decisions (e.g. approve design proposals).

@EdDev I agree that this would be better. I intendedly was a bit more vague, since I am not sure whether we can cover two distinct roles right now, given the number of contributors that we have currently that can and may want to fill such role(s).

Maybe it makes sense to combine them for now and if we see the need to separate the responsibilities we do that later? As of today I have the impression that qualified contributors would be able to fill both roles anyway.

Also there's still the possibility of delegation to the subproject leads. WDYT?

> 
> We already had one incident like this, when we had a volunteer who agreed to lead a SIG but was not yet a maintainer/approver. I do not intent to insist over this, but please be aware of the cost of unifying the two. We already have an issue with approvers managing to invest enough time in the technical area, adding more responsibility over it without an alternative will make their work harder.
> 
> [1] [kubernetes/community@`master`/committee-steering/governance/sig-governance.md#chair](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md?rgh-link-date=2024-09-24T11%3A29%3A36Z#chair) [2] [kubernetes/community@`master`/committee-steering/governance/sig-governance.md#tech-lead](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md?rgh-link-date=2024-09-24T11%3A29%3A36Z#tech-lead)

Reply:
I am a bit worried that we filter here members that can act as SIG chairs and are not approvers in the repos.

But if you accept this side-effect, we can proceed. If needed, we can reconsider it in the future with a real candidate.